### PR TITLE
Postgresql 12 build

### PR DESCRIPTION
**Pull request by Algunenano**
Here to pass tests.

https://trac.osgeo.org/postgis/ticket/4313
https://trac.osgeo.org/postgis/ticket/4312

### CODE REVIEW

File: raster/rt_pg/rtpg_mapalgebra.c [hunk 1]
```diff
@@ -147,7 +147,9 @@ static rtpg_nmapalgebra_arg rtpg_nmapalgebra_arg_init() {
 	arg->cextent = NULL;
 	arg->mask = NULL;
 
+#if POSTGIS_PGSQL_VERSION >= 120
```
Comment (laurenz) [inline raster/rt_pg/rtpg_mapalgebra.c]: Thanks for catching that omission.

**Comment by Algunenano** [comment]
Travis is running a week old PG 12 so the build is failing. @Komzpa @dbaston Any chance of updating it to HEAD to test this PR?

**Comment by dbaston** [comment]
@Algunenano I'll try to push an update. It's probably a good idea to get this run automatically on debbie or some other shared infrastructure; I don't really have interest to maintain this going forward.

**Comment by dbaston** [comment]
Updated PG12 on Travis; re-ran build.

**Comment by Algunenano** [comment]
> Updated PG12 on Travis; re-ran build.

Thanks a lot. Should we ask Regina for help to move this into one of the bots?

**Comment by Algunenano** [comment]
The tests for PG12 aren't passing but they work it you use  [`FUNC_MAX_ARGS`](https://github.com/Algunenano/postgis/commit/1e4a9f138309c87b8dc15eb1316fff5943b41199). I'll try to see which of the functions is using more parameters than declared.

**Comment by Algunenano** [comment]
> The tests for PG12 aren't passing but they work it you use FUNC_MAX_ARGS. I'll try to see which of the functions is using more parameters than declared.

After thinking about it, and looking at the code, I've decided to push it using `FUNC_MAX_ARGS` and leave the size optimization to whoever might be interested.

**Comment by Komzpa** [comment]
>After thinking about it, and looking at the code, I've decided to push it using FUNC_MAX_ARGS and leave the size optimization to whoever might be interested.

Please leave a ticket for "PostGIS Fund Me" milestone so knowledge of this deficiency is not lost.

**Pull request closed without merging**